basyx.examples.timerseries/src/main/java/org/eclipse/basyx/examples/timeseries/Executable.java

```diff
@@ -0,0 +1,153 @@
+ * 
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ * 
+ * SPDX-License-Identifier: MIT
+ ******************************************************************************/
+
+package org.eclipse.basyx.examples.timeseries;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import org.eclipse.basyx.aas.registration.api.IAASRegistry;
+import org.eclipse.basyx.aas.registration.proxy.AASRegistryProxy;
+import org.eclipse.basyx.submodel.metamodel.map.Submodel;
+import org.eclipse.basyx.submodel.restapi.SubmodelProvider;
+
+import javax.servlet.http.HttpServlet;
+
+import org.eclipse.basyx.aas.bundle.AASBundle;
+import org.eclipse.basyx.aas.manager.ConnectedAssetAdministrationShellManager;
+import org.eclipse.basyx.aas.metamodel.map.AssetAdministrationShell;
+import org.eclipse.basyx.aas.metamodel.map.descriptor.SubmodelDescriptor;
+import org.eclipse.basyx.vab.exception.provider.ProviderException;
+import org.eclipse.basyx.vab.factory.java.ModelProxyFactory;
+import org.eclipse.basyx.vab.modelprovider.VABElementProxy;
+import org.eclipse.basyx.vab.modelprovider.api.IModelProvider;
+import org.eclipse.basyx.vab.protocol.http.connector.HTTPConnectorFactory;
+import org.eclipse.basyx.vab.protocol.http.server.BaSyxContext;
+import org.eclipse.basyx.vab.protocol.http.server.BaSyxHTTPServer;
+import org.eclipse.basyx.vab.protocol.http.server.VABHTTPInterface;
+
+import org.eclipse.basyx.examples.timeseries.TimeSeriesSubmodel.TimeSeriesSubmodel;
+
+/*
+ * Builds an AirSensor AAS from the provided AirSensor.aasx
+ * Attaches a Submodel provider server with a TimeSeries
+ * operation
+ * 
+ * author Al-Obaidi
+ */
+public class Executable {
+    /**
+     * AAS endpoint configs
+     */
+    // private static final String REGISTRY_ENDPOINT =
+    // "http://localhost:4000/registry/";
+    // public static final String AASSERVER_ENDPOINT =
+    // "http://localhost:4001/aasServer";
+    private static final String REGISTRY_ENDPOINT = "http://registry:4000/registry/";
+    private static final String AASSERVER_ENDPOINT = "http://aas:4001/aasServer";
+    private static final int NUMBER_OF_RETRIES = 5;
+
+    private static final String AASX_PATH = "admin-shells/AirSensor.aasx";
+    private static final String AAS_ID_SHORT = "AirSensor";
+    private static final String TIMESERIES_SUBMODEL_ENDPOINT = "http://localhost:8002/AirSensor/TimeSeries";
+
+    private static final Logger logger = LoggerFactory.getLogger(Executable.class);
+
+    public static void main(String[] args) throws Exception {
```

Comment:
This method is too long, please refactor this main method and remove comments.
Also, not all the statements are on the same abstraction level.